### PR TITLE
Add unit tests for the Registrar.

### PR DESCRIPTION
**Pull request by DFurnes**
#### What's this PR do?
Just wanted to knock this out while waiting on PR review... this adds some simple unit tests for the Registrar's `resolve`, `resolveOrFail`, and `validateCredentials` methods.

#### How should this be reviewed?
Check that the tests seem to cover all expected behavior and edge cases for those functions!

#### Checklist
- [ ] Documentation added for changed endpoints.
- [x] Tests added for new features/bug fixes.

---
For review: @angaither @weerd

### CODE REVIEW

File: tests/RegistrarTest.php [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            'mobile' => '1 (123) 555-1234',
+        ]);
+
+        $this->assertSame($user->id, $resolvedUser->id);
+    }
+
+    /**
+     * Test that we return null if we can't conclusively resolve one account.
+     */
+    public function testResolveInconclusive()
```
Comment (weerd) [inline tests/RegistrarTest.php]: I like the name of this one :)

**Comment by weerd** [comment]
¡OLÉ! :sheep: it!
